src/smc-webapp/editor_chat.cjsx

```diff
@@ -74,7 +74,7 @@ underscore = require('underscore')
 misc = require('smc-util/misc')
 misc_page = require('./misc_page')
 {defaults, required} = misc
-{Markdown, TimeAgo, Tip} = require('./r_misc')
+{Markdown, TimeAgo, Tip, SetIntervalMixin} = require('./r_misc')
```

Comment:
is `SetIntervalMixin` still used? I have the impression there is no timer any more …